Modules/_testlimitedcapi/import.c

```diff
@@ -39,9 +39,11 @@ static PyObject *
 pyimport_getmodule(PyObject *Py_UNUSED(module), PyObject *name)
 {
     assert(!PyErr_Occurred());
+    NULLABLE(name);
     PyObject *module = PyImport_GetModule(name);
     if (module == NULL && !PyErr_Occurred()) {
-        Py_RETURN_NONE;
+        PyErr_SetString(PyExc_KeyError, "PyImport_GetModule() returned NULL");
```

Comment:
I meant returning PyExc_KeyError as a value, not raising an exception. How can you differentiate it from PyExc_KeyError raised in PyImport_GetModule()?

Reply:
Ah ok. I modified the code to return KeyError.